package Lesson2;

import java.io.IOException;

public class Task4 {
    public static void task1() throws IOException {
        System.out.println("Lesson 2 Task 4 exercise 1");
        Task4Task1.main(new String[]{});
    }

    public static void task2(){
        System.out.println("Lesson 2 Task 4 exercise 2");
        Task4Task2Sum.main(new String[]{});
    }

    public static void task3() throws IOException{
        System.out.println("Lesson 2 Task 4 exercise 3");
        Task4Task3.main(new String[]{});
    }

    public static void task4() throws IOException{
        System.out.println("Lesson 2 Task 4 exercise 4");
        Task4Task4Math.main(new String[]{});
    }

    public static void task5(){

    }

}
